run-directive/action.yml

```diff
@@ -0,0 +1,54 @@
+name: Run Directive
+run-name: test
+
+inputs:
+  hash:
+    description: 'Github commit hash to checkout'
+    required: false
+    default: ''
+  working-directory:
```

Comment:
Since the story specifies working directory and the goal is to run the directive on a specific component, I assume these are effectively the same thing.